### PR TITLE
Add method findInvalidStatementInDdlBatch to SQLCommand

### PR DESCRIPTION
**Pull request by wolffcm**
This pull request is pursuant to ongoing sqlcmd DDL batching work.

### CODE REVIEW

File: src/frontend/org/voltdb/utils/SQLCommand.java [hunk 1]
```diff
@@ -515,6 +518,39 @@ public static String readScriptFile(File file)
...
+    */
+   static Optional<Pair<Integer, String>> findInvalidStatementInDdlBatch(String batch) {
+
+       batch = Strings.nullToEmpty(batch);
+
+       List<String> stmts = SQLParser.parseQuery(batch);
+       int i = 0;
+       for (String stmt : stmts) {
+           boolean isDdl = SQLParser.queryIsDDL(stmt);
+           boolean isLoadOrRemoveClasses = SQLParser.isLoadOrRemoveClassesDirective(stmt);
```
Comment (jxchinaren) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: Why LOAD/REMOVE is special? It's not DDL, does the SQLParser.queryIsDDL handle it differently?

Comment (wolffcm) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: I don't know if it's DDL or not.  Does it belong in a batch?  If not, I can take it out.

Comment (jxchinaren) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: LOAD/REMOVE are the sqlcmd directives and not DDL statements that the server can understand.

Comment (paulmartel) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: The SQLParser.queryIsDDL only recognizes alter, create, drop, export, and partition.
The load/remove check is redundant.

Comment (paulmartel) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: Sorry. I misread the code. I meant to say that the load/remove check is wrong.
These are not ddl in any useful sense.

**Comment by wolffcm** [comment]
I pushed another commit.

File: src/frontend/org/voltdb/utils/SQLCommand.java (hunk 1 above)
Comment (paulmartel) [inline src/frontend/org/voltdb/utils/SQLCommand.java]: This code seems a bit bloated compared to the description in the ticket.
I thought that we had agreed that for the sake of expedience and consistency with direct calls to @AdHoc we were just going to check the first keyword in the batch and leave it to @AdHoc to enforce consistency and give a reasonable error message (for all callers, including non-ddl batch callers).

**Comment by wolffcm** [comment]
I misunderstood what the ticket was asking for.  I'll push another branch today.

**Pull request closed without merging**